app/models/date_range.rb

```diff
@@ -0,0 +1,30 @@
+class DateRange
+  attr_reader :start_date, :end_date
+  
+  # Step 1: only in use with Conference date range.
+  # Validated in Conference: presence and chronological order
+  #  because the input is done there.
+  
+  def initialize(start_date: start_date, end_date: end_date)
```

Comment:
Copy paste error *(or otherwise I don't understand the syntax)* 😉 

If you want to use required KWARGS, this should look like this:
```ruby
def initialize(start_date:, end_date:)
```